src/intercooler.js

```diff
@@ -8,7 +8,7 @@ var Intercooler = Intercooler || (function() {
 
   // work around zepto build issue TODO - fix me
   if((typeof Zepto !== "undefined") && ($ == null)) {
-    $ = Zepto
+    window["$"] = Zepto
```

Comment:
Good catch! Funny that this block was itself originally added as a hack for a build issue with Zepto.